build/style.css

```diff
@@ -100,3 +104,32 @@ strong {
 	opacity: 0.6;
 	border-bottom-color: #bb5536;
 }
+
+div.main.wrapper h2 {
+	color: #b30000;
+}
+
+div.wrapper h2.trouble {
+	float: right;
+	padding: 20px 0px;
+	color: #a9a9a9;
+	text-transform: uppercase;
+}
+div.article.full.section.content{
+	display: flex;
+}
+
+div.side-bar{
+	align-items: flex-end;
+	width: 40%;
+}
+
+div.table table{
+border: 1px black;	
+}
+
+div.footer {
```

Comment:
just call it `.footer` ... you only need to use `div.footer` if you also had a second element on the page that was a `<span>` or something like that and you wanted to style the 2 different types of elements that have the same class differently.